packages/test/test-driver-definitions/src/interfaces.ts

```diff
@@ -10,26 +10,31 @@ import {
 } from "@fluidframework/driver-definitions";
 
 /**
+ * Types of test drivers.
  * @public
  */
 export type TestDriverTypes = "tinylicious" | "t9s" | "routerlicious" | "r11s" | "odsp" | "local";
 
 /**
+ * Types of Routerlicious Endpoint.
  * @public
  */
 export type RouterliciousEndpoint = "frs" | "frsCanary" | "r11s" | "docker";
 
 /**
+ * Types of Odsp Endpoint.
  * @public
  */
 export type OdspEndpoint = "odsp" | "odsp-df";
 
 /**
+ * Types of Driver Endpoint.
  * @public
  */
 export type DriverEndpoint = RouterliciousEndpoint | OdspEndpoint;
 
 /**
+ * Interface for test drivers {@link @fluid-private/test-drivers#TinyliciousTestDriver}, {@link @fluid-private/test-drivers#RouterliciousTestDriver}, {@link @fluid-private/test-drivers#OdspTestDriver}, {@link @fluid-private/test-drivers#LocalServerTestDriver}.
```

Comment:
I appreciate the attempt to point to other relevant documentation here, but this seems like a layering violation to me. This is the base interface implemented by the items you pointed to. Generally, interfaces shouldn't know anything about their corresponding implementations. Instead, we should document the semantics of this interface on their own.

We can also disable the linter rule and leave this for the future.